fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -69,6 +69,8 @@ def __init__(self, estimator, constraints, eps=0.01, T=50, nu=None, eta_mul=2.0)
         self._best_t = None
         self._n_oracle_calls = 0
         self._oracle_calls_execution_time = None
+        self._lambda_vecs = pd.DataFrame()
```

Comment:
I think we will need to change the name `_lambda_vecs` to something else, but let's do that in a future PR.